src/query/filters.sparql

```diff
@@ -0,0 +1,25 @@
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX schema: <http://schema.org/>
+PREFIX cube: <https://cube.link/>
+PREFIX ssz: <https://ld.stadt-zuerich.ch/schema/>
+PREFIX hydra: <http://www.w3.org/ns/hydra/core#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+CONSTRUCT {
+  <urn:ssz:sets> a hydra:Collection ; hydra:member ?termSet .
+  ?termSet rdfs:label ?label
+}
+WHERE {
+  {
+    SELECT ?termSet ?label {
+      VALUES ?dimension { <{DIMENSION}> }
```

Comment:
I wonder if this functionality should be eventually in the rdf-cube-view-query library

Reply:
Curious. Please, elaborate what you have in mind

Reply:
To get the filters by the label is something that one could use for example in the cube-viewer. Now I think it's handled by Shapes. It's just something that popped up, not necessary now.